specification/elasticsan/resource-manager/Microsoft.ElasticSan/preview/2024-07-01-preview/elasticsan.json

```diff
@@ -2125,6 +2278,10 @@
         "enforceDataIntegrityCheckForIscsi": {
           "type": "boolean",
           "description": "A boolean indicating whether or not Data Integrity Check is enabled"
+        },
+        "deleteRetentionPolicy": {
```

Comment:
Why the "deleteRetentionPolicy" only on VG, but no on elasticSan?
Will the "deleteRetentionPolicy" works for both VG and volumn?
If customer what to use different "deleteRetentionPolicy" for VG and volumn, what can they do?

Reply:
We are providing the external soft delete option only at VG level. Yes, the deleteRetentionPolicy will work for both VG and volume. To update the retentionPolicy, customer can patch it in the VG's property.

Reply:
The description not updated to indicate this policy works for both volumngroup and volumn.

Reply:
will update this to "The retention policy for the soft deleted volume group and its associated resources"